google/string_utils.rb

```diff
@@ -48,5 +48,13 @@ def self.symbolize(key)
       key.to_sym unless key.nil?
     end
     # rubocop:enable Style/SafeNavigation
+
+    # Returns all the characters up until the period (.) or returns text
+    # unchanged if there is no period.
+    def self.first_sentence(text)
+      period_pos = text.index '.'
```

Comment:
Consider .?! as the standard stop-characters.

Reply:
Done